vshard/replicaset.lua

```diff
@@ -225,6 +231,13 @@ local function replicaset_connect_to_replica(replicaset, replica)
         conn:on_connect(netbox_on_connect)
         conn.on_disconnect_ref = netbox_on_disconnect
         conn:on_disconnect(netbox_on_disconnect)
+        -- Whether name validation is needed.
+        conn.name = replica.id == replica.name
```

Comment:
Please, set it before the triggers. Just to be sure that if the triggers would somehow run right away, they would see the flag.

Reply:
Sure, done